extensions/resource-deployment/src/test/ui/validation/validations.test.ts

```diff
@@ -0,0 +1,223 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as azdata from 'azdata';
+import 'mocha';
+import * as should from 'should';
+import * as sinon from 'sinon';
+import { createValidation, GreaterThanOrEqualsValidation, IntegerValidation, LessThanOrEqualsValidation, RegexValidation, validateInputBoxComponent, Validation, ValidationType, ValidationValueType } from '../../../ui/validation/validations';
+
+const inputBox = <azdata.InputBoxComponent>{
```

Comment:
Stuff like this is useful to have in a more generic place so other tests can easily find it

Reply:
I will refactor them out to a commonplace in the next test work that I am doing.